TemplePlus/critter.cpp

```diff
@@ -1810,6 +1815,56 @@ void LegacyCritterSystem::BuildRadialMenu(objHndl handle){
 		}
 	}
 }
+int LegacyCritterSystem::SetCritterAttacks(objHndl handle)
```

Comment:
This belongs in protos.cpp

Reply:
Funny, I first created in protos, but then tried to match the pattern, therefore moved here :)

Will fix, thanks